### ISSUE TITLE
Error: could not find module 'SwiftyTesseract' for target 'arm64-apple-ios-simulator'; 

### ISSUE THREAD

**Issue by MrFantuk** [khjde1207/tesseract_ocr#25]
Launching lib/main.dart on iPhone 11 in debug mode...
lib/main.dart:1
Xcode build done.                                            3,1s
Failed to build iOS app
Error output from Xcode build:
↳
    objc[14554]: Class AMSupportURLConnectionDelegate is implemented in both /usr/lib/libauthinstall.dylib (0x206e46b90) and /Library/Apple/System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x1068942c8). One of the two will be used. Which one is undefined.
    objc[14554]: Class AMSupportURLSession is implemented in both /usr/lib/libauthinstall.dylib (0x206e46be0) and /Library/Apple/System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x106894318). One of the two will be used. Which one is undefined.
    ** BUILD FAILED **
Xcode's output:
↳
    Command CompileSwiftSources failed with a nonzero exit code
/Users/denis/Desktop/development/programms/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_tesseract_ocr-0.4.19/ios/Classes/SwiftFlutterTesseractOcrPlugin.swift:3:8: error: could not find module 'SwiftyTesseract' for target 'arm64-apple-ios-simulator'; found: x86_64-apple-ios-simulator, x86_64, at: /Users/denis/Desktop/development/apps/nmo_helper/build/ios/Debug-iphonesimulator/SwiftyTesseract/SwiftyTesseract.framework/Modules/SwiftyTesseract.swiftmodule
    import SwiftyTesseract
           ^

Also removing from 
dev_dependencies:
  ...
  flutter_tesseract_ocr:
 projects works.

**Comment by khjde1207** [comment]
https://github.com/khjde1207/tesseract_ocr/issues/16
Can you please check if this method solves it?

**Comment by ammaarkhan** [comment]
Hi, I am still having this issue when attempting to run the example code mentioned in pub.dev. The issue link you mentioned issue doesn't help. Has anyone figured out the fix?

**Comment by gigafunk** [comment]
I have this problem, i think its a matter of mot having arm64 , ie M1, apple silicone, support in the ios library this uses

**Comment by tranquockhanh0506** [comment]
I also have the same problem, any one fixed?

**Comment by ammaarkhan** [comment]
Hey @tranquockhanh0506, I couldn't get it to work, ended up using this lib: https://github.com/naveenvignesh5/rn-ocr-lib. Hope it helps.

**Comment by tranquockhanh0506** [comment]
the lib only support react-native right? @ammaarkhan

**Comment by ammaarkhan** [comment]
yeah.. I'm not sure I understand your question. Did you click on the link? @tranquockhanh0506

**Comment by tranquockhanh0506** [comment]
l clicked, but the lib you suggest me only support react native, but l want flutter :(((( @ammaarkhan

**Comment by ammaarkhan** [comment]
Oh! So sorry! I thought this library was for react native! Our team was exploring flutter but we ended up switching to react native. Apologies! @tranquockhanh0506

**Comment by khjde1207** [comment]
I don't know how to solve it either.
If there is someone who is good at it, please help them.

**Comment by tranquockhanh0506** [comment]
@khjde1207  Have you solved it yet? :)))

**Comment by gideonoyekale** [comment]
Has anyone solved this yet? Still having the issue please

**Comment by satet-dev** [comment]
I encountered the same problem and was able to bypass it by _excluding the arm64 architecture in the build configurations_. Locate the `Podfile` in your XCode project (if you don't have one, try running your Flutter project and see if that changes anything; it will likely be under the `Pods` icon on the left menu in your XCode project). Flutter has something akin to the following placed at the end of the file:

```ruby
post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)
  end
end
```

**Change these lines in your `Podfile` to the following:**

```ruby
post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)
    target.build_configurations.each do |config|
      config.build_settings['EXCLUDED_ARCHS[sdk=iphonesimulator*]'] = 'arm64'
    end
  end
end
```

This should fix any errors that arise when using an iOS simulator with this library with M1 or M2 chips. Thanks.
